src/main/java/core/basesyntax/Car.java

```diff
@@ -1,5 +1,78 @@
 package core.basesyntax;
 
-public class Car {
+public class Car implements Cloneable {
+    private static final String LINE_BREAKER = "\n";
+    private static final String DOT = ".";
+    private String typeOfCar;
```

Comment:
```suggestion
public class Car implements Cloneable {
    private String typeOfCar;
```
In model classes we don't usually create constants for new line and dots, unless they may be reused in several methods (not only toString())